packages/utilities/psammead-locales/moment/pa.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
+const moment = require('moment');
+
+moment.defineLocale('pa-in', {
```

Comment:
Hey :) Curious why youve gone with `defineLocale` here, vs `updateLocale` like we do with yoruba? I think updating it such that the [`postFormat`](https://github.com/moment/moment/blob/develop/locale/pa-in.js#L80) is undefined, should stop the numbers being translated :)

Reply:
ahh that seems like a much better approach. Will update now :)